README.md

```diff
@@ -42,7 +42,9 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+// for (i=0; i<foods.length; i++){
+  console.log ("I want to eat a [fruit]")
+}
```

Comment:
Please review our suggestions in the solution branch.  Get with me if you still have questions.